docs/qiita-16S-analysis.rst

```diff
@@ -3,218 +3,424 @@
 
 
 
-Analysis of Closed Reference processing
----------------------------------------
+Analysis of Closed Reference Process
+------------------------------------
 
-To create an analysis, select `Create new analysis` from the top menu.
+To create an analysis, select "Create new analysis" from the top menu.
 
 This will take you to a list of studies with samples available to you for
-analysis, divided between your studies and publically available ('Other')
+analysis, divided between your studies and publically available ("Other")
 studies.
 
-.. figure::  images/analysis-studies-page.png
+.. figure::  images/analysis_studies_page.png
    :align:   center
 
 Find the study you created for this tutorial under "Your Studies". Click the
```

Comment:
As of today's update it is now labelled "Public Studies" instead of Other Studies